webui/react/src/components/Bar.tsx

```diff
@@ -19,10 +20,19 @@ export interface Props {
 }
 
 const partStyle = (part: BarPart) => {
-  return {
+  let style = {
     backgroundColor: part.color,
+    borderColor: 'var(--theme-colors-monochrome-11)',
+    borderStyle: 'none',
+    borderWidth: 1,
     width: floatToPercent(part.percent, 0),
   };
+
+  if(part.borded){
```

Comment:
hmm, why isn't lint erroring out on this line?
overwhelming majority of our code has a space in `if (` - I believe we should be enforcing this